python/CMakeLists.txt

```diff
@@ -48,11 +64,19 @@ declare_mlir_python_extension(TTMLIRPythonExtensions.Main
     TTModule.cpp
     TTKernelModule.cpp
     Overrides.cpp
+    Passes.cpp
   EMBED_CAPI_LINK_LIBS
     MLIRCAPITransforms
     TTMLIRCAPI
   PRIVATE_LINK_LIBS
     LLVMSupport
+    ${dialect_libs}
+    ${conversion_libs}
+    MLIR
+    TTMLIRStatic
+    TTMLIRConversions
+    TTMLIRTTNNToEmitC
+    TTNNTargetFlatbuffer
```

Comment:
I'm a little worried about these linked libraries, we'll see if CI is cool with it.

Reply:
CI Seems to be fine so far, seemingly the passes can run with only TTMLIRStatic included.